lib/airbrake/better_errors.rb

```diff
@@ -0,0 +1,38 @@
+require 'better_errors'
+
+module BetterErrors
+  class Middleware
+    alias_method :show_error_page_original, :show_error_page
+
+    private
+
+    def show_error_page(env, exception=nil)
+      env['airbrake.error_id'] = notify_airbrake(env, exception)
+      show_error_page_original(env, exception)
+    end
+
+    def notify_airbrake(env, exception)
+      unless ignored_user_agent? env
+        Airbrake.notify_or_ignore(exception, request_data(env))
+      end
+    end
+
+    def ignored_user_agent?(env)
```

Comment:
I wonder why this logic is not a part of `Airbrake.notify_or_ignore` method? Like `Airbrake.notify_or_ignore` method can check `user_agent` string and ignore provided exception.